src/toil/job.py

```diff
@@ -472,6 +472,10 @@ def makeString(x):
         # the logging has been captured to be reported on the leader.
         self.logJobStoreFileID = None
 
+        # Every time we update a job description in place in the job store, we
+        # increment this.
```

Comment:
Is this only for debugging?  The purpose of this isn't clear to me.

Reply:
I added this for debugging because I thought we might have been forgetting to save an update or running the wrong version of a job somewhere; it turned out that wasn't the case and we were giving the same version of the job to two batch systems, but I left this in because I want to use it later when poking around the leader looking for race conditions with its weird job update system.